### PR TITLE
31 clean up pd rule

### PR DESCRIPTION
**Pull request by jccf19**
Update pd_rule

### CODE REVIEW

**Comment by jhabriel** [comment]
@gusespinola : I will take care of reviewing this PR.

File: src/pd_rule.m [hunk 1]
```diff
@@ -1,8 +1,55 @@
-function miter = pd_rule(m, minitial, mmin, ...
-                         res, iter, mstep, mmax, alpha, delta)
+function miter = pd_rule(m, mInitial, mMin, res, iter, ...
+                         mStep, mMax, alphaP, alphaD)
     % Rule for the Proportional-Derivative law.
```
[the remark this answers is not in the document]
Comment (jccf19) [reply src/pd_rule.m:3]: Done

File: src/pd_rule.m [hunk 2]
```diff
@@ -1,8 +1,55 @@
...
     %
-    %   TODO: Complete rest of the docstring.
+    %   %   Signature:
+    %   ----------
+    %
+    %   miter = pd_rule(m, mInitial, mMin, ...
+    %                    res, iter(size(iter, 1), :), ...
```
[the remark this answers is not in the document]
Comment (jccf19) [reply src/pd_rule.m]: Done.

File: src/pd_rule.m [hunk 3]
```diff
@@ -25,51 +72,36 @@
...
-        %     miter=miter-mstep;
-        % mj=mmax-mstep;
-        mj = mmax;
+    if mj > mMax
+        mj = mMax;
     end
 
-    miter = [mj minitial];
-    miter = [mj minitial ap ad];
+    miter = [mj mInitial];
```
[the remark this answers is not in the document]
Comment (jccf19) [reply src/pd_rule.m:108]: Donde

File: src/pd_rule.m [hunk 4]
```diff
@@ -25,51 +72,36 @@
...
-        % mj= m + round(alpha*(res(iter,:)/res(iter-1,:)) + delta ...
-        % * ((res(iter,:) - res(iter-2,:))/(2*res(iter-1,:))));
 
-        ap = res(iter, :) / res(iter - 1, :);
-        ad = (res(iter, :) - res(iter - 2, :)) / (2 * res(iter - 1, :));
 
     elseif iter > 2
-        mj = m + ceil(alpha * (res(iter, :) / res(iter - 1, :)));
-        %    mj= m + round(alpha*(res(iter,:)/res(iter-1,:)));
+        mj = m + ceil(alphaP * (res(iter, :) / res(iter - 1, :)));
```
[the remark this answers is not in the document]
Comment (jccf19) [reply src/pd_rule.m:89]: Done.

File: src/pd_rule.m [hunk 5]
```diff
@@ -25,51 +72,36 @@
...
-    ap = 0;
-    ad = 0;
+
 
     if iter > 3
 
         mj = m + ceil( ...
-                      alpha * (res(iter, :) / res(iter - 1, :)) + ...
-                      delta * ( ...
+                      alphaP * (res(iter, :) / res(iter - 1, :)) + ...
```
[the remark this answers is not in the document]
Comment (jccf19) [reply src/pd_rule.m:81]: Done.

[the remark this answers is not in the document]
Comment (jccf19) [reply src/pd_rule.m:81]: Done.

File: src/pd_rule.m [hunk 6]
```diff
@@ -1,8 +1,55 @@
-function miter = pd_rule(m, minitial, mmin, ...
```
[the remark this answers is not in the document]
Comment (jccf19) [reply src/pd_rule.m:1]: Done.

**Review by jhabriel — APPROVED** [review]
Great work @jccf19!

**Pull request merged**